browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -90,6 +90,11 @@
         </settings-radio-group>
       </div>
     </div>
+    <settings-toggle-button id="cosmeticFilteringControlType"
+        pref="{{prefs.brave.cosmetic_filtering_default}}"
+        label="$i18n{cosmeticFilteringControlLabel}"
```

Comment:
Could you put a screenshot for how this looks in the PR? Thanks!

Reply:
Will need to take a look at the design and especially strings for this.

Reply:
FWIW, I based the implementation from [this mockup](https://app.goabstract.com/projects/bad752b0-846f-11e8-aef1-25c293be1275/branches/d7aacab1-fa2f-4adf-a0db-3b0eb43098ec/commits/413fc3f9a676ccca135f229fc3aac16a84b9e258/files/47C7B326-36ED-428B-A70C-006F108E6A50/layers/D93DE291-D528-4CC4-B6A7-115DFF48E61E?mode=design)

Reply:
Posted in a separate comment below.